src/NuGet.Core/NuGet.ProjectModel/JsonPackageSpecReader.Utf8JsonStreamReader.cs

```diff
@@ -852,10 +851,6 @@ private static void ReadMSBuildMetadata(ref Utf8JsonStreamReader jsonReader, Pac
                     {
                         packagesPath = ExtractMacro(jsonReader.ReadNextTokenAsString(), userSettingsDirectory, useMacros);
                     }
-                    else if (jsonReader.ValueTextEquals(ProjectJsonPathPropertyName))
```

Comment:
there's some test failures related to this.

Reply:
Decided to just revert this as it's kind of tangent to the primary goal here. 
By the way, I don't have or need this property within VS as we already obtain the project.json path before calling the migrator. In case you noticed it's missing in the new spec reader.